consensus/chronology/chronology.go

```diff
@@ -116,6 +138,33 @@ func (chr *chronology) RemoveAllSubrounds() {
 	chr.mutSubrounds.Unlock()
 }
 
+func (chr *chronology) resetChronologyStuckAlarm() {
+	duration := chr.rounder.TimeDuration() * numRoundsToWaitBeforeSignalingChronologyStuck
+	alarmID := "chronologyStuck"
+
+	chr.alarmScheduler.Cancel(alarmID)
+
+	cb := func(alarmID string) {
+		buffer := new(bytes.Buffer)
+		err := pprof.Lookup("goroutine").WriteTo(buffer, 1)
+		if err != nil {
+			log.Error("could not dump goroutines")
+		}
+		log.Debug(buffer.String())
+
+		arg := endProcess.ArgEndProcess{
+			Reason:      "chronology is stuck",
+			Description: "startRound() was not called before the alarm expired",
+		}
+		chr.chanStopNodeProcess <- arg
+
+		time.Sleep(core.TimeToGracefullyCloseNode)
+		os.Exit(1)
```

Comment:
The closing process is handled by the main function. This os.Exit should be removed from this position as it makes the code harder to debug in case it early exists by some reason. Imagine we have n structures each calling od.Exit and the node suddenly stops. Which one did the call?

Reply:
Removed.